docs/software/system_stats.md

```diff
@@ -566,118 +566,186 @@ License: [Apache 2.0](https://github.com/prometheus/node_exporter/blob/master/LI
     
-	You may bookmark this and save it as your browser's home page, or alternatively look into tools like nginx proxy manager and a dns server such as adguard home to give it a nice internal domain name such as homer.box.
-     
-=== "Configuration"
+    You may bookmark this and save it as your browser's home page, or alternatively look into tools like **Nginx Proxy Manager** and a DNS server such as **AdGuard Home** to give it a nice internal domain name such as `homer.box`.
 
-	To configure entries and icons for homer, you can edit it's config.yml file 
+=== "Configuration"
 
- 	![Homer configuration screenshot](../assets/images/homer-config.png){: width="400" height="218" loading="lazy"}
- 
-	In the screenshot above, a column titled selfhosted is being made which contains 4 entries. In the same directory as the config.yml i've made the icons2 folder which contains icon images
+    To configure entries and icons for Homer, you can edit it's `config.yml` file.  
+    Example contents:
+
+    ```
+    services:
+      - name: "Selfhosted"
+        icon: "fas fa-cloud"
+        items:
+          - name: "Adguard Home"
+            logo: "assets/icons2/adguard.png"
+            subtitle: "A blackhole for the internet"
+            tag: "Selfhosted"
+            url: "https://adguard.box"
+            target: "_blank"
+          - name: "Nginx"
+            logo: "assets/icons2/nginx.png"
+            subtitle: "Reverse proxy - load balancer"
+            tag: "Selfhosted"
+            url: "https://nginx.box"
+            target: "_blank"
+          - name: "Shiori"
+            logo: "assets/icons2/shiori.png"
+            subtitle: "Web archiving + Read later"
+            tag: "Selfhosted"
+            url: "https://shiori.box"
+            target: "_blank"
+          - name: "Nextcloud"
+            logo: "assets/icons2/nextcloud.png"
+            subtitle: "A google suite alternative"
+            tag: "Selfhosted"
+            url: "https://nextcloud.box"
+            target: "_blank"
+    ```
  
+    In the example above, a column titled `Selfhosted` is being made which contains 4 entries. In the same directory as the `config.yml`, an `icons2` folder exists, which contains icon images.
+
 === "Update"
 
-	To update homer simply re-run the installer from `dietpi-software`, then `cd /mnt/dietpi_userdata/homer_backup` and replace the files you previously changed in /var/www/homer with the files from your backup. 
+    To update Homer simply re-run the installer from 
     
-    For e.g, for your config.yml, it'd be:
-    1) `rm /var/www/homer/assets/config.yml`
-    2) `mv config.yml /var/www/homer/assets`
-	
+    ```sh
+    dietpi-software reinstall 205
+    ```
+    
+    then 
+    
+    ```sh
+    cd /mnt/dietpi_userdata/homer_backup
+    ``` 
+    
+    and replace the files you previously changed in `/var/www/homer` with the files from your backup. 
+    
+    For e.g, for your `config.yml`, it would be:
```

Comment:
`config.yml` is preserved automatically, which should be instead noted here explicitly. As an example to restore from backup, `/var/www/homer/assets/manifest.json` can be used.

Reply:
@MichaIng: Do you mean a text like this?

---

To update Homer simply re-run the installer by executing

```
dietpi-software reinstall 205
```

Remark: The `dietpi-software` script preserves the previous `config.yml` automatically, there is no necessity for the user to do a restore manually.

If other files in `/var/www/homer` were previously changed by the user, they need to be copied in place from `/mnt/dietpi_userdata/homer_backup`. As an example for a changed file `manifest.json` this could be done via:

```sh
cd /mnt/dietpi_userdata/homer_backup
mv manifest.json /var/www/homer/assets
```

Reply:
Yes, sounds good 👍.